web-client/src/app/views/wallet/wallet.page.ts

```diff
@@ -39,13 +39,16 @@ export class WalletPage implements OnInit {
   balances: Observable<AssetAmount[]> = this.sessionQuery.allBalances;
 
   /** Enable the "Send Money" action if KYC status is either cleared or not required. */
-  actionSendMoneyEnabled: Observable<boolean> =
-    this.sessionQuery.onfidoCheckIsClear.pipe(
-      map(
-        (onfidoCheckIsClear: boolean) =>
-          onfidoCheckIsClear || !this.requireKycBeforeSendPayment
-      )
-    );
+  actionSendMoneyEnabled: Observable<boolean> = combineLatest(
+    this.sessionQuery.onfidoCheckIsClear,
+    this.sessionQuery.allBalances
```

Comment:
It might be good to rather use `this.balances` here to explicitly align with the displayed balances, in case we later start implementing functionality to filter and show only show a subset of all available balances?

```suggestion
    this.balances
```

Reply:
I thought about using `this.balances` instead. The reason I ended up going with the one from state is because we want a default balance shown if there is no balance in `this.sessionQuery.allBalances` and the way to implement that might be to set a default value for `this.balances`. 

If we later want to have some filtered balances I also think that it might be better add a query for it on `SessionQuery` instead, and update the usage.